sips/sip-015/sip-015-network-upgrade.md

```diff
@@ -0,0 +1,1582 @@
+# Preamble
+
+SIP Number: 015
+
+Title: Stacks Upgrade of Proof-of-Transfer and Clarity
+
+Authors:
```

Comment:
The authors list can absolutely be expanded.  You don't have to be a developer to contribute to a SIP's activation, especially one as broad in scope as this one!  I know I've missed people who deserve to be here, so please speak up and get credit where credit is due :)

Reply:
For example, @Hero-Gamer should definitely be on this list, but I need an email address to put down :)

Reply:
came here because of @Hero-Gamer so +1 to this suggestion

Reply:
@niclaz do I know you?? What name are you on discord or twitter lol.. thanks for your vouch!.. Lots of great interesting things happens & knowledge on Github def a great place to explorer Stacks! 
Not sure if I should be on the list, but if you guys feel appropriate, my email is: herogamerthesht572@gmail.com

Reply:
Through Twitter @Hero-Gamer, you posted the link to this SIP and it inspired me to read it and comment - I'm @niclaztweets on that platform

Reply:
I'd love to see Jason @whoabuddy on, made my comment in the other tab reasons stated there: https://github.com/stacksgov/sips/pull/95/files#r1016619773

Reply:
Yup, added!